docs/How-We-Work/our-workflow.md

```diff
@@ -48,20 +48,21 @@ Issues will be created in the [Raft TDP repo](https://github.com/raft-tech/TANF-
   - documentation necessary to review the work is captured
   - A pull request is drafted using the [PR template]() for issues that will result in changes to the documentation/code managed in the TDP repo. **See [ADR 009]() for more details on the PR review process, which may vary depending on the issue type (e.g. frontend, security controls, dependencies)**.
 - If all this is in place, the assignee should move the card to “Raft Review” column/pipeline in Zenhub and ping the appropriate reviewer(s).
-     - Once _research_ issues are approved in raft review, the assignee should tag the PO for review and move the issue/PR to the "QASP Review" pipeline in Zenhub. 
+     - Once _research_ issues are approved in raft review, the assignee should tag the PO for review and move the issue/PR to the "QASP Review" pipeline in Zenhub.
      - For _design_ issues, the assignee should tag at least one developer for raft review. Upon approval,  the reviewer tags the ACF Tech Lead and moves the issue/PR to the "QASP Review" pipeline in Zenhub.
-     - _Dev_ issues must proceed through the **[tabletop process](https://github.com/raft-tech/TANF-app/tree/develop/docs/How-We-Work/Developer-Tabletops.md)** and be approved in raft review before it is moved the "QASP Review" pipeline in Zenhub. 
+     - _Dev_ issues must proceed through the **[tabletop process](https://github.com/raft-tech/TANF-app/tree/develop/docs/How-We-Work/Developer-Tabletops.md)** and be approved in raft review before it is moved the "QASP Review" pipeline in Zenhub.
+- When approved and QASP review is not required, the assignee should move the card to the "Ready to Merge" column/pipeline in Zenhub.
 
-### The issue is ready for QASP review
+### The issue needs QASP review (Not a requirement)
```

Comment:
it would be helpful to distinguish when this is/isn't a requirement.

Reply:
For QASP, I review/test branches in sequence, against QASP-approved code. I do this to help catch issues before it gets to staging. If QASP review now shifts to staging, is raft review happening against everything raft-approved? For example, if PR A is raft-approved before PR B, will PR B always be reviewed and tested after PR A is merged into its base branch?